webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -192,7 +227,11 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
   const [error] = useState(false);
   const [canceler] = useState(new AbortController());
 
-  const colorMap = useGlasbey(settings.selectedExperiments);
+  const colorMap = useGlasbey(
+    experiments.map((loadable) => {
+      return Loadable.getOrElse(undefined, loadable)?.experiment.id ?? 0;
+    }),
+  );
```

Comment:
does this return colors that match the comparison view?

Reply:
to be fair, the root of the issue is the grid selection updating out of sync with the id selection -- while both are called at the same time, the call to update the settings store happens out of the stack. The [selection consolidation PR](https://github.com/determined-ai/determined/pull/8860) fixes this by using the settings store directly, so i'd recommend de-scoping this fix

Reply:
understood, thanks, will do.